src/main/ts/ScriptLoader.ts

```diff
@@ -48,16 +52,23 @@ const CreateScriptLoader = (): ScriptLoader => {
     }
   };
 
-  const load = (doc: Document, url: string, callback: callbackFn) => {
+  const load = (doc: Document, url: string, callback: callbackFn, loadTimeout?: number, scriptAttributes?: IProps['scriptAttributes']) => {
+
+    function scriptTagInjection() {
```

Comment:
We don't typically use named functions like this instead preferring to use anon functions that are assigned to a constant:
```
const scriptTagInjection = () => {
```